src/main/java/com/google/sps/servlets/CommentService.java

```diff
@@ -0,0 +1,77 @@
+package com.google.sps.servlets;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.CommentThread;
+import com.google.api.services.youtube.model.CommentThreadListResponse;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.lang.Exception;
```

Comment:
nit: import should follow ascii order (yeah I didn't know this exists either) 

https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

Reply:
Didn't know this, thanks!